src/main/resources/openapi/audiorecording.yaml

```diff
@@ -0,0 +1,130 @@
+openapi: 3.0.1
+info:
+  title: Audio Recording Service
+  version: v0.1.0
+servers:
+  - url: http://localhost:8080
+paths:
+  /audios:
+    post:
+      tags:
+        - audio-recording
+      summary: Upload an audio recording
+      operationId: addAudio
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AddAudioRequest'
+      responses:
+        200:
+          description: Recording accepted
+        400:
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetail'
```

Comment:
Change to the following to use the "common" schema definition for RFC-7807 Problem Details (also note the change to the content type to `json+problem`:
```yaml
          content:
            application/json+problem:
              schema:
                $ref: './problem.yaml'
```

Reply:
ACCEPT